my-dsl.md

```diff
@@ -2,37 +2,67 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+I have found a language called [TikZ](https://sourceforge.net/projects/pgf/). It's a syntax layer for the PGF package for LaTeX, and makes using it easier.
+
 
 # Domain
 _Describe the language's domain in five words._
 
+Make cool graphics in LaTeX.
+
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+A TikZ "program" will generally be part of a larger LaTeX (or other TeX) document, and will compile along with it. A TikZ program within a LaTeX document is written inside a tikzpicture environment, and will create graphics such as a diagram of a finite-state machine in the document that LaTeX outputs.
```

Comment:
How does the code you write get computed? This just describes where it is written.